<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudLinkerClient</class>
 <widget class="QWidget" name="CloudLinkerClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CloudLinkerClient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="regist_page">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: #3fbcbc;
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>REGIST</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QtMaterialTextField" name="rigist_uesrname_lineEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QtMaterialTextField" name="regist_passward_lineEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QtMaterialTextField" name="regist_passward_ag_lineEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="scrolltext" name="Tips_regist" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QtMaterialRaisedButton" name="return_login_pushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>返回</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtMaterialRaisedButton" name="regist_pushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="login_page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
          <widget class="QtMaterialTextField" name="passward_lineEdit">
           <property name="styleSheet">
            <string notr="true">font-size: 14pt</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QtMaterialRaisedButton" name="go_regist_pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>去注册</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QtMaterialRaisedButton" name="login_pushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoRepeatDelay">
              <number>300</number>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QtMaterialTextField" name="user_lineEdit">
           <property name="styleSheet">
            <string notr="true">font-size: 14pt</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Normal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QWidget" name="container" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="scrolltext" name="Tips" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: #3fbcbc;
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>LOGIN</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: #D3D3D3;
}</string>
           </property>
           <property name="text">
            <string>未连接</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scrolltext</class>
   <extends>QWidget</extends>
   <header>scrolltext.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialtextfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
